src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1810,7 +1810,21 @@ public function insert_posts( $items = [] ) {
 							}
 
 							if ( empty( $venue_unique_field ) || ( $lookup_venues_by_title && empty( $venue ) ) ) {
-								$venue = get_page_by_title( $event['Venue']['Venue'], 'OBJECT', Tribe__Events__Venue::POSTTYPE );
+								$venue_query = new WP_Query(
+									[
+										'post_type'      => Tribe__Events__Venue::POSTTYPE,
+										'title'          => $event['Venue']['Venue'],
+										'post_status'    => 'all',
```

Comment:
Is it `all` or `any` ? or both ?

Reply:
Ah! You're right. It's `any`, not `all`. Great catch! I'll fix it first thing in the AM.